packages/guideline-blocks-settings/src/hooks/useAttachments.tsx

```diff
@@ -1,6 +1,9 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { AppBridgeBlock, Asset, useBlockAssets } from '@frontify/app-bridge';
+import { type ReactNode, createContext, useContext } from 'react';
+
+import { type BlockProps } from '../';
```

Comment:
Avoid this kind of path and be more precise

Reply:
fixed